1001-2000/LIT2000Mazmur.xml

```diff
@@ -7299,6 +7299,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      ለገብርከ ፡ እስመ ፡ ትእዛዘከ ፡ ኢረሳዕኩ ።<lb n="3"/> </l>
                </ab>
             </div>
+            
+            <div type="textpart" subtype="Psalmus" xml:id="Ps118Spiritual">
+               <label>Psalm 118 including the Spritual Meaning of the Hebrew Letters</label>
+               <note>This ID is to be used in cases where Psalm 118 contains, in a sentence after the name of the letters,
```

Comment:
I would comment this out, so that it is in the file, but not part of the edition.

Reply:
Do you mean the note or the entire textpart?

Reply:
yep